src/main/java/club/issizler/okyanus/api/json/JsonMessage.java

```diff
@@ -0,0 +1,24 @@
+package club.issizler.okyanus.api.json;
+
+import club.issizler.okyanus.api.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public interface JsonMessage {
+
+    /**
+     * Sends the message to online players
+     */
+    void sendAll();
```

Comment:
Are we absolutely sure these should be in the message? Is it the message's responsibility to send itself?

Reply:
The messages must send own message itself. The opp logic is like that.